src/test/java/com/ixi_U/chatbot/service/ChatBotServiceTest.java

```diff
@@ -0,0 +1,31 @@
+package com.ixi_U.chatbot.service;
+
+import static com.ixi_U.chatbot.constants.TestConstants.CHATBOT_WELCOME_MESSAGE;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.junit.jupiter.MockitoExtension;
+import reactor.core.publisher.Flux;
+import reactor.test.StepVerifier;
+
+@ExtendWith(MockitoExtension.class)
+class ChatBotServiceTest {
+
+    @InjectMocks
+    ChatBotService chatBotService;
+
+    @Test
+    @DisplayName("웰컴 메세지를 응답한다")
+    public void getWelcomeMessage() {
+
+        //given & when
+        Flux<String> welcomeMessage = chatBotService.getWelcomeMessage();
```

Comment:
```suggestion
        //given

        //when
        Flux<String> welcomeMessage = chatBotService.getWelcomeMessage();
```
차라리 `given`절을 비어두는 것이 어떨까요? fixture 데이터가 필요없는데 given & when을 묶어두는 것은 독자로 하여금 혼동을 줄 수 있을 것 같습니다.

Reply:
수정하겠습니다